File: LISTA_01/ex.java
import java.util.Scanner;

public class ex {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.print("Digite seu nome: ");
        String nome = entrada.nextLine();

        System.out.println("Olá, " + nome + "!");

        entrada.close();
    }
}

// javac ex.java

File: LISTA_01/ex.cpp
#include <iostream>

using namespace std;

int main()   {
    string nome;
    cout << "Digite seu nome: ";
    cin >> nome;
    cout << "Olá, " << nome << endl;
    return 0;  
}

// COMPILAR: g++ ex.cpp -o ex
// EXECUTAR: ./ex